legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/ingestmode/UnitemporalDeltaAbstract.java

```diff
@@ -79,6 +84,15 @@ public Void visitMaxVersionStrategy(MaxVersionStrategyAbstract maxVersionStrateg
                 {
                     throw new IllegalStateException("Cannot build UnitemporalDelta, MergeDataVersionResolver is mandatory for MaxVersionStrategy");
                 }
+
+                MergeDataVersionResolver mergeStrategy = maxVersionStrategy.mergeDataVersionResolver().get();
```

Comment:
Can be moved to a common code as repeated in line 107-114

Reply:
sure, changed!